app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/AMLSController.scala

```diff
@@ -234,6 +234,11 @@ class AMLSController @Inject()(
       Ok(amlsNumberNotFoundTemplate())
     }
   }
+  def showAmlsDateNotMatched: Action[AnyContent] = Action.async { implicit request =>
+    withSubscribingAgent { _ =>
+      Ok(amlsNumberNotFoundTemplate())
```

Comment:
isn't this the same view as `showAmlsNumberNotFound` ? is it meant to be the new page instead

Reply:
Good spot I'll change it now